Lesson_5/Homework_5.py

```diff
@@ -0,0 +1,29 @@
+# palindrome
+string = input("write whatever you want: ")
+result = string == string[::-1]
+print(f'your string is palindrome - {result}')
+
+# list to string & to list
+list_1 = ['Hillel', 'AQA', 'TEST']
+list_to_str = list_1[0] +', ' + list_1[1] +', ' + list_1[2]
+str_to_list = list_to_str.split(', ')
+print(f'list to string - {list_to_str} - {type(list_to_str)} \nstring back to list - {str_to_list} - {type(str_to_list)}')
+
+
+# # Sort list
+list_1 = ['Tst', 'aBc', 'TEST', 'Hello', 'neW']
+list_1.sort(key=str.lower)
+print(f'Sorted list is {list_1}')
+
+
+# # List inside of list
+list_1 = [1, 2, 3, 4, 5, 6, [1, 2, 3, ['Win']]]
+list_win = list_1[6][3][0].split(',')
+print(f'New list = {list_win}')
```

Comment:
This can be simpler
list_1 = [1, 2, 3, 4, 5, 6, [1, 2, 3, ['Win']]]
list_win = list_1[6][3][0]
print(f'New list = {[list_win]}')


If we add only one element in list, we can use [element]